modules/schema/src/main/resources/lucuma/odb/graphql/OdbSchema.graphql

```diff
@@ -3084,11 +3089,13 @@ type ObservationEdit { #implements Event {
   """
   editType: EditType!
 
+  observationId: ObservationId!
+
   """
   Edited object
```

Comment:
maybe add `(if not deleted)`?  It is the case that when deleted this is `null` correct?